vendor.conf

```diff
@@ -27,7 +27,7 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            f5a55a9516d1c6e2ade9bec22b83259caeed3a84 
+github.com/moby/buildkit                            v0.6.1
```

Comment:
Not a blocker but I just want to note for the future that I prefer commits over tags and tags can always be added as a comment. cc @thaJeztah

Reply:
see my comment above https://github.com/moby/moby/pull/39745#discussion_r314147781 😇